test/setupFile.js

```diff
@@ -18,8 +18,6 @@ global.document.createRange = () => ({
 });
 global.document.getSelection = () => {};
 
-window.MutationObserver = function () {};
-window.MutationObserver.prototype.observe = function () {};
 window.HTMLElement.prototype.scrollIntoView = jest.fn();
```

Comment:
😬 I'm guessing jsdom now has MutationObserver ?

Reply:
I realized I was no longer using MutationObserver in my code, so i didn't need to mock it.